Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ruby:2.7
+
+# Add yarn apt repository
+# https://classic.yarnpkg.com/en/docs/install#debian-stable
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
+
+# Install system packages
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+      build-essential \
+      imagemagick \
+      libarchive-dev \
+      libffi-dev \
+      libmagickwand-dev \
+      libpq-dev \
+      libsasl2-dev \
+      libxml2-dev \
+      libxslt1-dev \
+      locales \
+      nodejs \
+      default-jre-headless \
+      phantomjs \
```

Comment:
We don't use phantomjs any more, see #2544 and #2598 

Without geckodriver, the test suite doesn't work. I've had a look at packages, but I could only find [a request on debian to include geckodriver](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=8742070 and it hasn't had much activity. Perhaps someone with better search-fu can find a debian package for geckodriver?

Reply:
This is pretty far outside my experience, so hopefully someone can chime in.